src/Routes.tsx

```diff
@@ -1,30 +1,35 @@
 import React, { FunctionComponent } from "react";
-import { Switch, Route } from "react-router";
+import { Route, Switch } from "react-router";
 import { BrowserRouter } from "react-router-dom";
+import PrivateRoute from "./components/accessories/privateRoute/PrivateRoute";
 import DashboardActivity from "./components/activities/dashboardActivity/DashboardActivity";
+import LoginActivity from "./components/activities/loginActivity/LoginActivity";
 import NewPatientActivity from "./components/activities/newPatientActivity/NewPatientActivity";
-import SearchPatientActivity from "./components/activities/searchPatientActivity/SearchPatientActivity";
 import PatientDetailsActivity from "./components/activities/patientDetailsActivity/PatientDetailsActivity";
+import SearchPatientActivity from "./components/activities/searchPatientActivity/SearchPatientActivity";
 
 const Routes: FunctionComponent = () => {
   return (
     <BrowserRouter basename={process.env.PUBLIC_URL}>
       <Switch>
-        <Route path="/dashboard">
+        <Route exact path="/">
```

Comment:
I would prefer if / was assigned to dashboard, and /login to login

Reply:
I opened the task [OP-320](https://openhospital.atlassian.net/browse/OP-320) for doing this change further.